### PR TITLE
Edit styles.

### PR DESCRIPTION
**Pull request by franclobo**
- Edit Styles.

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -1,26 +1,162 @@
...
+}
+
+header {
+  display: flex;
+  justify-content: flex-start;
   align-items: center;
+  gap: 20px;
   color: var(--title);
+  margin: 20px 5%;
 }
```
Comment (roshan-bajgain) [inline src/index.css]: - [x] Kindly use the logo on the right side of the header to make it similar to the Figma design
### Logo
logo

File: src/index.css [hunk 2]
```diff
@@ -1,26 +1,162 @@
...
   --link: #121212;
+  --panel: #fafafa;
+}
+
+.single-book,
+.category {
+  background: var(--panel);
+  margin: 20px 5%;
+  box-shadow: 8px 8px 25px 0 rgba(0, 0, 0, 0.5);
+}
```
Comment (roshan-bajgain) [inline src/index.css]: - [x] You have missed this two-part `update progress` and `completed`. Kindly implement it
### Design
figma

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @franclobo, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- PR has a good title and summary. ✔️

## Required Changes ♻️

- [x] When I add a book it does not show on the page, I have to refresh the page to have the title and author of the book. Kindly fix the issue as well.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by EricMbouwe — APPROVED** [review]
Hi @franclobo, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**